proto/pilab/cli/v1/cli_common.proto

```
@@ -133,12 +107,12 @@ message Network {
   google.protobuf.Timestamp updated_at = 10;
 }
 
-enum NetworkState {
-  NETWORK_STATE_UNSPECIFIED = 0;
-  NETWORK_STATE_CREATING = 1;
-  NETWORK_STATE_READY = 2;
-  NETWORK_STATE_ERROR = 3;
-  NETWORK_STATE_DELETING = 4;
+enum VSwitchState {
+  VSWITCH_STATE_UNSPECIFIED = 0;
```

Buf (1.71.0)

[error] Enum value name "VSWITCH_STATE_UNSPECIFIED" should be prefixed with "V_SWITCH_STATE_".

(ENUM_VALUE_PREFIX)